File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-04/plagiarized/L4/09/Level4.ts
/**
 * @author FD8DBE9073AFCC0504CD6901E1091CAD
 */
function tabelKonversi(): void {
    console.log("Miles\t\tKilometers");
    console.log("-------------------------------");

    let mil: number = 1;
    while (mil <= 10) {
        console.log(`${mil}\t\t${mil * 1.609}`);
        mil++;
    }
}

tabelKonversi();

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-04/plagiarized/L4/09/Level4.kt
package IR

/**
 * @author FD8DBE9073AFCC0504CD6901E1091CAD
 */
object Level4 {

    @JvmStatic
    fun main(args: Array<String>) {
        println("Miles\t\tKilometers")
        println("-------------------------------")

        tabelKonversi()
    }

    private fun tabelKonversi() {
        var mil = 1
        while (mil <= 10) {
            println("$mil\t\t${mil * 1.609}")
            mil++
        }
    }
}

